### PR TITLE
DIV-6907: Trying to fix sign out feature tests

### PR DESCRIPTION
**Pull request by jvictorcf**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by yarpo — APPROVED** [review]

**Review by madjava — APPROVED** [review]

File: config/development_template.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # ---
-# Used when running the app locally.
-# Rename this file to `development.yaml`.
-# This file is used for any development config which should not be version controlled.
-# For example, you can add a LaunchDarkly SDK here to test locally and it will not be pushed to Git.
+# Used when running the app locally., make a copy of this file a name it `development.yaml`.
```
Comment (jacmurphy) [inline config/development_template.yaml:2]: Small gramatical error - ".,"

File: test/end-to-end/paths/logout.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ Scenario.skip('Logout on Sign Out', function (I) {
   I.haveRespondentAddress();
   I.haveMarriageCert();
   I.readFinancialRemedy();
-  I.retry(1).signOut();
+  I.retry(3).signOut();
```
Comment (jacmurphy) [inline test/end-to-end/paths/logout.js:39]: Out of curisolity, do we have any metrics to show how many retries tend to lead to successes?

**Review by jacmurphy — APPROVED** [review]

File: test/end-to-end/paths/logout.js (hunk 2 above)
Comment (jvictorcf) [reply test/end-to-end/paths/logout.js:39]: I don't think so.

File: config/development_template.yaml (hunk 1 above)
Comment (jvictorcf) [reply config/development_template.yaml]: Done. Good spot.

**Pull request merged**